.github/workflows/semantic-checker.yml

```diff
@@ -0,0 +1,63 @@
+name: MageOS Semantic Checker
+run-name: ${{ github.actor }} is checking the semantic version
+on: [workflow_call]
+jobs:
+  semantic-version:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Environment Debugging Output
+        run: |
+          echo "Event_path: ${GITHUB_EVENT_PATH}"
+          echo 'Event: ${{ toJSON(github.event.pull_request) }}'
+      # TODO: Add pull request labels
+      # - name: Clean up labels
+      #   uses: andymckay/labeler@master
+      #   with:
+      #     remove-labels: "svc"
+      - name: Check out Semamtic Version Checker tool
+        uses: actions/checkout@v3
+        with:
+          repository: mage-os/mirror-magento-semver
+          path: magento-semver
+      - name: Check out base branch - ${{ github.base_ref }}
+        uses: actions/checkout@v3
+        with:
+          path: base-branch
+          ref: ${{ github.base_ref }}
+      - name: Check out new changes - ${{ github.event.pull_request.head.repo.full_name }} / ${{ github.head_ref }}
+        uses: actions/checkout@v3
+        with:
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
+          path: this-change
+          ref: ${{ github.head_ref }}
+      - name: Install Composer
+        uses: php-actions/composer@v6
+        with:
+          command: help
```

Comment:
Is `help` the correct command or debugging code?

Reply:
> Is `help` the correct command or debugging code?

The action installs Composer, which is needed for installing the SVC tool, and then runs `composer install` by default, but in this case I don't think we want to install MageOS anywhere, so this was the most simple way I could think of to skip running a Composer command. Any thoughts?

Reply:
That makes sense. How about adding a comment that explains this?
ii probably won't be the last one stumbling over it.

Reply:
I (finally) added a comment line for this